service/notifier.go

```diff
@@ -272,6 +283,10 @@ func (p *notifier) sendMessageToWebhook(notification *Notification, webhook *con
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != http.StatusOK {
```

Comment:
in this method, let's also enforce the `CallbackRequestTimeout` config.

Reply:
added the timeout in http client creation